tests/Cargo.toml

```diff
@@ -29,6 +29,9 @@ walkdir = "2.4.0"
 anyhow = "1.0.79"
 image = { version = "0.24.8", default-features = false, features  = ["png"] }
 futures = "0.3.30"
+env_logger = "0.10.1"
+tracing = { workspace = true}
```

Comment:
This is missing a space, but it was like that before, oh well.

Reply:
Fixed